pkgs/development/python-modules/surt/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, six
+, tldextract
+, pytestCheckHook
+, pytest-cov
+}:
+
+buildPythonPackage rec {
+  version = "0.3.1";
+  pname = "surt";
+
+  src = fetchFromGitHub {
+    owner = "internetarchive";
+    repo = "surt";
+    rev = "6934c321b3e2f66af9c001d882475949f00570c5"; # Has no git tag
+    sha256 = "sha256-pSMNpFfq2V0ANWNFPcb1DwPHccbfddo9P4xZ+ghwbz4=";
+  };
+
+  propagatedBuildInputs = [
+    six
+    tldextract
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+    pytest-cov
+  ];
+
+  checkPhase = ''
+    runHook preCheck
+
+    py.test -v --cov surt tests/
+
+    runHook postCheck
+  '';
```

Comment:
```suggestion
  pytestFlagsArray = [ "-v" "--cov surt" ];
```

Reply:
I just removed `checkPhase` entirely since the extra output is not really needed. But I didn't know about `pytestFlagsArray` before, so thank you.